src/Aspire.Dashboard/Components/ResourcesGridColumns/StateColumnDisplay.razor

```diff
@@ -0,0 +1,36 @@
+﻿@using Aspire.Dashboard.Model
+@using Aspire.Dashboard.Otlp.Model
+@using Aspire.Dashboard.Resources
+@inject IStringLocalizer<Columns> Loc
+
+<div class="resource-state-container">
+    @if (Resource is { State: ResourceStates.ExitedState /* containers */ or ResourceStates.FinishedState /* executables */, ExitCode: not null and not 0 })
```

Comment:
I stopped some containers, and I could see the icon:

![image](https://github.com/dotnet/aspire/assets/303201/2d515850-ea75-480f-9b2f-ef0c9edefe53)

But I also killed processes for the projects and they continue to be reported as running and no status icon is displayed. Do you know why that is? Is there an issue with Aspire not detecting that processes have been killed?

Reply:
That project behavior is not reproducing for me

Reply:
Hmm ok. Maybe I'm using an old version of DCP. It won't be related to this PR.